assets/app/components/site/editor/editorContainer.js

```diff
@@ -22,7 +22,7 @@ const propTypes = {
 };
 
 const redirectToFileOnBranch = (siteId, branch, filePath) => {
-  routeActions.redirect(`/sites/${siteId}/edit/${branch}/${filePath}`);
+  routeActions.redirect(`/sites/${siteId}/edit/${branch}/${filePath}`, 'replace');
```

Comment:
Can we have two different functions in routeActions? One for push and one for replace? In this manner we can think about what people who call each function in routerAction really mean without having to expose the particular 'push' or 'replace' concept to outsiders.

Naturally, my first question as a client would be "What are the valid methods? Are push and replace the only ones? When would I use each of these methods? What happens if I have a typo in my method?"